### DISCUSSION TITLE
第4节课笔记(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#258] [笔记]
第4节课笔记(13班)

**Comment by SiliconSisyphus** [comment]
https://www.notion.so/60f62dcdd0a548beacc03a02969d5ac2

**Comment by qingyin741** [comment]
https://zhuanlan.zhihu.com/p/677931238

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/MmHWwBChUizAABk8NJCcw5t5nTf?from=from_copylink

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135597127
